test/constants.ts

```diff
@@ -0,0 +1,54 @@
+import { DEFAULT_CONFIG_STORE_VERSION } from "../src/clients";
+import { randomAddress, toWei, originChainId, destinationChainId, repaymentChainId } from "./utils";
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+export const CONFIG_STORE_VERSION = DEFAULT_CONFIG_STORE_VERSION;
+export const randomL1Token = randomAddress();
+export const randomOriginToken = randomAddress();
+export const randomDestinationToken = randomAddress();
+export const randomDestinationToken2 = randomAddress();
+
+// This lookback of 24 hours should be enough to cover all Deposit events in the test cases.
+export const DEFAULT_UNFILLED_DEPOSIT_LOOKBACK = 1 * 24 * 60 * 60;
+
+// Max number of refunds in relayer refund leaf for a { repaymentChainId, L2TokenAddress }.
+export const MAX_REFUNDS_PER_RELAYER_REFUND_LEAF = 3;
+
+// Max number of L1 tokens for a chain ID in a pool rebalance leaf.
+export const MAX_L1_TOKENS_PER_POOL_REBALANCE_LEAF = 3;
+
+export const BUNDLE_END_BLOCK_BUFFER = 5;
+
+// DAI's Rate model.
+export const sampleRateModel = {
+  UBar: toWei(0.8).toString(),
+  R0: toWei(0.04).toString(),
+  R1: toWei(0.07).toString(),
+  R2: toWei(0.75).toString(),
+};
+
+export const defaultTokenConfig = JSON.stringify({
+  rateModel: sampleRateModel,
+});
+
+// Add Mainnet chain ID 1 to the chain ID list because the dataworker uses this chain to look up latest GlobalConfig
+// updates for config variables like MAX_REFUND_COUNT_FOR_RELAYER_REPAYMENT_LEAF.
+export const CHAIN_ID_TEST_LIST = [originChainId, destinationChainId, repaymentChainId, 1];
+export const DEFAULT_BLOCK_RANGE_FOR_CHAIN = [
+  // For each chain ID in above list, default range is set super high so as to contain all events in a test
+  // in the straightforward test cases.
+  [0, 1_000_000],
+  [0, 1_000_000],
+  [0, 1_000_000],
+  [0, 1_000_000],
+];
+
+export const IMPOSSIBLE_BLOCK_RANGE = DEFAULT_BLOCK_RANGE_FOR_CHAIN.map((range) => [range[1], range[1]]);
+
+export const baseSpeedUpString = "ACROSS-V2-FEE-1.0";
```

Comment:
With the understanding that this is a pre-existing constant, I'm not sure whether this is correct after the SpokePool update that modified the way we verify signatures. @nicholaspai wdyt?

https://github.com/across-protocol/contracts-v2/blob/456d48015acda7f4f8426f43d994a364f1b0fca8/contracts/SpokePool.sol#L265

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/408/commits/eb1f8c14ac3ac1b26e4d8cb7e5a932eb4d37327a